### PR TITLE
Error handle for empty resource scan

### PR DESCRIPTION
**Pull request by rcohencyberarmor**
https://cyberarmor-io.atlassian.net/browse/SUB-1097

### CODE REVIEW

File: core/cautils/fileutils_test.go [hunk 1]
```diff
@@ -30,8 +37,16 @@ func TestListFiles(t *testing.T) {
 	assert.Equal(t, 12, len(files))
 }
 
+func TestLoadResourcesFromFilesEmptyFiles(t *testing.T) {
```
Comment (Daniel-GrunbergerCA) [inline core/cautils/fileutils_test.go:40]: `LoadResourcesFromFilesEmptyFiles` is not a function. This is an specific test case for the  `LoadResourcesFromFiles` function. As such, should be under `TestLoadResourcesFromFiles`

File: core/cautils/fileutils_test.go [hunk 2]
```diff
@@ -21,6 +23,11 @@ func helmChartPath() string {
 	return filepath.Join(filepath.Dir(o), "..", "examples", "helm_chart")
 }
 
+func emptyDirectory() string {
```
Comment (Daniel-GrunbergerCA) [inline core/cautils/fileutils_test.go:26]: why can't we just use the path?

File: core/cautils/fileutils.go [hunk 3]
```diff
@@ -97,22 +102,22 @@ func LoadResourcesFromKustomizeDirectory(ctx context.Context, basePath string) (
...
-func LoadResourcesFromFiles(ctx context.Context, input, rootPath string) map[string][]workloadinterface.IMetadata {
+func LoadResourcesFromFiles(ctx context.Context, input, rootPath string) (map[string][]workloadinterface.IMetadata, error) {
 	files, errs := listFiles(input)
 	if len(errs) > 0 {
 		logger.L().Ctx(ctx).Warning(fmt.Sprintf("%v", errs))
 	}
 	if len(files) == 0 {
 		logger.L().Ctx(ctx).Error("no files found to scan", helpers.String("input", input))
-		return nil
+		return nil, errors.New(fmt.Sprintf(ErrNoFilesToScan, input))
```
Comment (Daniel-GrunbergerCA) [inline core/cautils/fileutils.go:112]: pls mind the linter comments

Comment (rcohencyberarmor) [reply core/cautils/fileutils.go:112]: I don't see any linter issue.

Comment (rcohencyberarmor) [reply core/cautils/fileutils.go:112]: image
can you be more spevcific?

Comment (Daniel-GrunbergerCA) [reply core/cautils/fileutils.go:112]: image
You can see them on your commit, for example: https://github.com/kubescape/kubescape/pull/1296/commits/b730ef51543e9f7d84db6137870f1454c38a104e

**Review by Daniel-GrunbergerCA — APPROVED** [review]

**Review by dwertent — APPROVED** [review]

**Pull request merged**